cookbook/index.rst

```diff
@@ -9,6 +9,9 @@ This book contains list of most regularly used solutions in OroPlatform.
   * :doc:`/cookbook/entities/creating_entities`
   * :doc:`/cookbook/entities/data_fixtures`
   * :doc:`/cookbook/entities/datagrid`
+  * :doc:`/cookbook/entities/managing_tasks`
+  * :doc:`/cookbook/entities/deleting_tasks`
```

Comment:
Previous links was about all entities, and this two links describes only tasks. I think it will be better to name they just `managing_entities` and `deleting_entities`. And this files should be start with `For example, let's take a task entity...`